src/services/tourettes.js

```diff
@@ -62,12 +62,22 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
```

Comment:
`isNumber` would maybe be a better name since there is no type `Integer` in JavaScript